workspace.code-workspace

```diff
@@ -95,7 +98,7 @@
     "eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }],
     "task.autoDetect": "off",
     "editor.codeActionsOnSave": {
-      "source.fixAll.eslint": true
+      "source.fixAll.eslint": "explicit"
```

Comment:
What does this do?

Reply:
I don't know, I think you commited this change https://github.com/stryker-mutator/stryker-js/pull/4644/commits/ad98b5c37f200db4d444536cfd8d871c7ca072ad